src/main/java/com/uid2/admin/store/MultiScopeStoreWriter.java

```diff
@@ -49,9 +61,19 @@ private Map<Integer, T> getCurrentState(Collection<Integer> siteIds) throws Exce
         return currentState;
     }
 
-    private void write(Map<Integer, T> desiredState, Collection<Integer> sitesToWrite, JsonObject extraMeta) throws Exception {
-        for (Integer addedSite : sitesToWrite) {
-            factory.getWriter(addedSite).upload(desiredState.get(addedSite), extraMeta);
+    public void uploadEncrypted(Map<Integer, T> desiredState, JsonObject extraMeta) throws Exception {
```

Comment:
Can you rename this to uploadPrivateWithEncryption to match the uploadPublicWithEncryption